src/data/script/ConstraintChangerModPlugin.java

```diff
@@ -642,6 +661,17 @@ private float safeUnboxing(Float object) {
 
             return retVal;
         }
+
+        private double safeUnboxing(Double object) {
+            double retVal;
+            if (object == null) {
+                retVal = 0;
+            } else {
+                retVal = object;
+            }
+
+            return retVal;
+        }
```

Comment:
useless method. delete it since the backing fields in `CyberneticAugmentation` class are floats and we'd need to downcast the `double` to a `float`, introducing loss of precision and making the whole thing oxymoronic.
`CyberneticAugmentation.BONUS_PER_ELITE_SKILL` is a float.

the method is unused. get rid of it.

Reply:
done. resolving.